File: Informatyka sem 2 2016/Labolatorium 10 Zadanie 1-2/Zadanie1/src/zadanie1/VerboseObject.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package zadanie1;

import static java.lang.System.out;

/**
 *
 * @author Lenovo
 */
public class VerboseObject 
{
 private String name;
 public VerboseObject(String name)
 {
     this.name=name;
     out.println("Wywołałes konstruktor obiektu o nazwie: "+name);
 }
protected void finalize()
 {
     out.println("Wywołałes destruktor obiektu o nazwie: "+name);
 }

    void saySomething() 
    {
        out.println("Hello");
    }

}

File: Informatyka sem 2 2016/Labolatorium 2 Zadanie 1/Labolatorium 2 Zadanie 1/VerboseObject.cpp
#include "VerboseObject.h"
#include<iostream>
#include<string>
using namespace std;
VerboseObject::VerboseObject()
{
	cout << "Konstruktor domyslny\n";
}
VerboseObject::VerboseObject(string n) :name(n)
{
	cout << "Konstruktor obiektu " << name << "\n";
}
VerboseObject::~VerboseObject()
{
	cout << "Destruktor obiektu " << name << "\n";
}
void VerboseObject::Metod()
{
	cout << "Metoda obiektu " << name << "\n";
}
